src/sentry/notifications/platform/templates/seer.py

```diff
@@ -0,0 +1,92 @@
+from dataclasses import dataclass
+
+from sentry.notifications.platform.registry import template_registry
+from sentry.notifications.platform.types import (
+    CodeTextBlock,
+    NotificationCategory,
+    NotificationData,
+    NotificationRenderedTemplate,
+    NotificationTemplate,
+    ParagraphBlock,
+)
+
+# TODO(leander): We're going to shortcut this with a custom renderer, since not all the
+# required SlackBlocks are available in the NotificationRenderedTemplate.
+
+
+@dataclass
+class SeerAutofixTrigger(NotificationData):
+    source: str = "seer-autofix-trigger"
+    label: str = "Start RCA"
+
+
+@template_registry.register(SeerAutofixTrigger.source)
+class SeerAutofixTriggerTemplate(NotificationTemplate[SeerAutofixTrigger]):
+    category = NotificationCategory.SEER
+    example_data = SeerAutofixTrigger(source="seer-autofix-trigger")
+    hide_from_debugger = True
+
+    def render(self, data: SeerAutofixTrigger) -> NotificationRenderedTemplate:
+        return NotificationRenderedTemplate(subject="Seer Autofix Trigger", body=[])
```

Comment:
oo do we have images for what these templates look like ? I'm curious how the bodyless template will present